packages/react-charting/src/components/AreaChart/AreaChart.base.tsx

```diff
@@ -249,14 +249,14 @@ export class AreaChartBase extends React.Component<IAreaChartProps, IAreaChartSt
     let axisType: XAxisTypes | null = null;
```

Comment:
Perhaps, you can assign during initilization:
const axisType: XAxisTypes = getTypeOfAxis(lineChartData![0].data[0].x, true);

Reply:
Thanks for the suggestion. This has been incorporated